### PR TITLE
[TDV-674] Update price attributes on asset pair manager

### PR DESCRIPTION
**Pull request by ardier16**
+ help hints

### CODE REVIEW

File: src/components/User/Assets/AssetPairs/components/AssetPairManager.vue [hunk 1]
```diff
@@ -153,6 +217,9 @@ export default {
     },
 
     async updatePolicy () {
```
Comment (max-nes) [inline src/components/User/Assets/AssetPairs/components/AssetPairManager.vue]: actually, not only policies being updated

File: src/components/common/fields/TickField.vue [hunk 2]
```diff
@@ -146,7 +164,7 @@ export default {
   min-width: 1.6rem;
   height: 1.6rem;
   margin: 0.2rem 1.3rem 0 0;
-  z-index: 0;
+  // z-index: 0;
```
Comment (max-nes) [inline src/components/common/fields/TickField.vue]: typo?

Comment (ardier16) [reply src/components/common/fields/TickField.vue]: Yes, fixed

File: src/components/User/Assets/AssetPairs/components/AssetPairManager.vue (hunk 1 above)
Comment (ardier16) [reply src/components/User/Assets/AssetPairs/components/AssetPairManager.vue]: Renamed

File: src/components/User/Assets/AssetPairs/components/AssetPairManager.vue [hunk 3]
```diff
@@ -34,38 +34,98 @@
...
             label="Is tradable"
-            title="Allowed to trade this pair on secondary market"
             :cb-value="ASSET_PAIR_POLICIES.tradeableSecondaryMarket"
-          />
-          <!-- eslint-disable max-len -->
+            show-help
+          >
+            <span class="asset-pair-manager__tip-message">
+              Allowed to trade this pair on secondary market
+            </span>
```
[the remark this answers is not in the document]
Comment (ardier16) [reply src/components/User/Assets/AssetPairs/components/AssetPairManager.vue]: Fixed

File: src/components/User/Assets/AssetPairs/components/AssetPairCreator.vue [hunk 4]
```diff
@@ -40,7 +40,18 @@
...
+          show-help
+        >
+          <p class="asset-pair-creator__tip-message">
+            <span>
+              The correction of physical price in percents.
+              If physical price and restriction by physical price are set,
+              minimum price for this pair offer will be
+            </span>
+            <strong>physicalPrice * physicalPriceCorrection</strong>
+          </p>
```
[the remark this answers is not in the document]
Comment (ardier16) [reply src/components/User/Assets/AssetPairs/components/AssetPairCreator.vue]: Got it, fixed

**Pull request merged**